src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -4,6 +4,7 @@
 $fa-font-path: '../node_modules/fork-awesome/fonts';
 @import '../node_modules/fork-awesome/scss/fork-awesome';
 $font_family_1: 'ForkAwesome';
+@import 'app/ceph/dashboard/info-card/info-card-popover.scss';
```

Comment:
Could this be imported in the info-card or health component?

Reply:
> Could this be imported in the info-card or health component?

@tspmelo The fact is that when popover had large text, it got hidden by below info card (conflicting CSS). As the elements belong to different parents, the only solution that I found to work was this:
https://valor-software.com/ngx-bootstrap/#/popover#container-body
As the popover belongs to body, I had to apply its style in styles.scss (but keeping them in a separate file).